### PR TITLE
Refining bobaeditor.md

### PR DESCRIPTION
**Pull request by cyrusae**
Bringing into alignment with current readme refinements for the boba-editor repo.

- Added explanation of forking before cloning
- Added link to GitHub's explanation of forking
- Made explanation of what to do about the terminal more detailed (will most people need that? no; can it be reassuring for people who are panicking about task startup? hopefully)
- Added explanation of how to use the desktop GUI instead, because it's genuinely good for routine stuff okay

### CODE REVIEW

File: docs/engineering/start-developing/bobaeditor.md [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
 
 \[[code](https://github.com/essential-randomness/boba-editor), [demo](https://bobaeditor.netlify.app/?path=/story/editor-preview--simple-editor)]
 
 BobaEditor is BobaBoard's own extension of the [QuillJS text editor](https://quilljs.com/), and is responsible for anything related to **text formatting** and **embeds** (both in "edit" and "display" mode).
 
-## Install Instructions
+## Check Out the Codebase 
 
-In a terminal, run the following commands:
+First, [fork the boba-editor repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) in order to save a copy of the code to your own account.
```
Comment (essential-randomness) [inline docs/engineering/start-developing/bobaeditor.md:13]: If you add these, then the clone command won't be `https://github.com/essential-randomness/boba-editor.git` but `https://github.com/YOUR_GITHUB_ACCOUNT/boba-editor.git`

File: docs/engineering/start-developing/bobaeditor.md [hunk 2]
```diff
@@ -21,7 +23,9 @@ cd boba-editor
 yarn install
 ```
 
-## Development Instructions
+(If you would prefer to use a GUI with GitHub, you can [fork and clone](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop) via GitHub Desktop, then enter your new directory in a terminal and run `yarn install`.)
```
Comment (essential-randomness) [inline docs/engineering/start-developing/bobaeditor.md:26]: You should use a "tip" callout for this.

:::tip
your text
:::

**Review by essential-randomness — CHANGES_REQUESTED** [review]

File: docs/engineering/start-developing/bobaeditor.md (hunk 2 above)
Comment (cyrusae) [reply docs/engineering/start-developing/bobaeditor.md]: Done!

File: docs/engineering/start-developing/bobaeditor.md (hunk 1 above)
Comment (cyrusae) [reply docs/engineering/start-developing/bobaeditor.md:13]: oh right yes

**Review by essential-randomness — APPROVED** [review]

**Pull request merged**